.github/workflows/nightly_run.yml

```diff
@@ -4,6 +4,14 @@
 name: nightly_run
 
 on:
+  pull_request:
+    types:
+      - opened
```

Comment:
Will this CI be triggered when each PR is opened and updated?

Reply:
Seems OK, I missed the `paths:` expression, L13.
Can refer to other CI for how it is triggered:
https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/.github/workflows/compilation_on_android_ubuntu.yml#L13-L15